mix.exs

```diff
@@ -5,7 +5,7 @@ defmodule Gringotts.Examples.Mixfile do
     [
       app: :gringotts_examples,
       version: "0.1.0",
-      elixir: "~> 1.5.2",
+      elixir: "~> 1.5.1",
```

Comment:
That's a very restrictive [`Version`](https://hexdocs.pm/elixir/Version.html). I'm sure this will work with older versions of elixir too.
Gringotts has this `elixir: >= 1.3.0`
@pkrawat1: Need help with choosing/testing against versions.

Reply:
@oyeb yes, we have to support almost all widely used versions have to spend some time on it. Testing is easier on travis. they support multiple version testing AFAIK.